convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,435 @@
+            "#800000",  # brown
+        )
+
+        result = {
+            "name": category,
+            "type": type_of_categories,
+            "id": category,
+            "metadata": {"color": random.choice(possible_categories_colors)},
+        }
+        return result
+
+    def request_annotations_to_post_categories(self) -> List:
+        post_categories_url = f"{settings.annotation_service_url}categories/"
+        LOGGER.debug(
+            "Making requests to url: %s to post annotations",
+            post_categories_url,
+        )
+        pages_objs = self.badgerdoc_format.badgerdoc_annotation.pages[0].objs
+        categories_of_type_box = {
+            pages_obj.category for pages_obj in pages_objs
+        }
+        categories_of_type_link = self.get_categories_of_links(pages_objs)
+        all_categories = {
+            "box": categories_of_type_box,
+            "link": categories_of_type_link,
+        }
+
+        for (
+            type_of_categories,
+            set_of_categories_names,
+        ) in all_categories.items():
+            for category in set_of_categories_names:
+                post_categories_body = self.form_post_category_body(
+                    type_of_categories, category
+                )
+                LOGGER.debug(
+                    "Making request to post categories with this request body: %s",
+                    post_categories_body,
+                )
+
+                try:
+                    request_to_post_categories = requests.post(
+                        url=post_categories_url,
+                        headers=self.request_headers,
+                        json=post_categories_body,
+                    )
+                    request_to_post_categories.raise_for_status()
+                except requests.exceptions.RequestException as e:
+                    if request_to_post_categories.status_code == 400 and request_to_post_categories.json() == {
+                        "detail": "Field constraint error. Category id must be unique."
+                    }:
+                        LOGGER.warning(
+                            "Category %s already exists in annotation. Skipping this request",
+                            category,
+                        )
+                        continue
+
+                    LOGGER.exception(
+                        "Failed request to 'annotation' to post categories for converted annotations"
+                    )
+                    raise HTTPException(
+                        status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                        detail="Failed request to 'annotation' to post categories for converted annotations",
+                    ) from e
+
+                LOGGER.debug(
+                    "Got this response from annotation service: %s",
+                    request_to_post_categories.json(),
+                )
+
+        return list(categories_of_type_box) + list(categories_of_type_link)
```

Comment:
Not a big deal, but this would work a bit faster :) 
```suggestion
        return [*categories_of_type_box, *categories_of_type_link]
```

Reply:
Done